setup/config/config_address.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"io/fs"
+	"net/url"
+)
+
+const (
+	NetworkTcp  = "tcp"
```

Comment:
Idiomatic Go would have this be `NetworkTCP`.

Reply:
fixed